sites/aviationpros.com/server/components/nodes/content-card.marko

```diff
@@ -24,6 +24,7 @@ $ const { linkAttrs } = input;
       fluid=true
       ar="16:9"
       width=(input.imageWidth || 300)
+      ...input.image
```

Comment:
Detect if an image is present. If not, add the `zoomed` modifier via `modifiers=["zoomed"]`